capa/main.py

```diff
@@ -125,18 +136,27 @@ def find_capabilities(ruleset, extractor, disable_progress=None):
         # to disable progress completely
         pbar = lambda s, *args, **kwargs: s
 
-    for f in pbar(list(extractor.get_functions()), desc="matching", unit=" functions"):
+    functions = list(extractor.get_functions())
+
+    for f in pbar(functions, desc="matching", unit=" functions"):
+        function_address = f.__int__()
```

Comment:
this is still a little clumsy

Reply:
agree. i think i was running into trouble using `int(f)` but maybe thats fixed in py3. will check.

Reply:
fixed here: 9ca1a7e 

this commit needs review by @mr-tz @mike-hunhoff

Reply:
looks great!